lib/std/crypto/gimli.zig

```diff
@@ -61,7 +61,7 @@ pub const State = struct {
     }
 
     pub fn squeeze(self: *Self, out: []u8) void {
-        var i = usize(0);
+        var i = @as(usize, 0);
```

Comment:
I think a better idiom would be `var i: usize = 0`

Reply:
Agreed. They are semantically identical, therefore the shorter one is preferred.